src/components/auth/ProfileModal/ProfileModal.tsx

```diff
@@ -40,6 +41,7 @@ export default function ProfileModal({
 }: ProfileModalProps) {
   // Prefetch telegram linked account data
   getLinkedTelegramAccountsQuery.useQuery({ address })
+  useAnalytics().sendEvent('open_account_menu')
```

Comment:
I think this one will be called multiple times, because its called every render, should be called inside useEffect and use if (props.isOpen)

or call the event in the action that triggers the opening of the modal

Reply:
Resolved